lib/ui/settings/network/network_screen.dart

```diff
@@ -266,7 +284,7 @@ class _NetworkScreenState extends ConsumerState<NetworkScreen> {
                   ).copyWith(bottom: 64.h),
                   child: WnFilledButton.icon(
                     visualState: WnButtonVisualState.secondary,
-                    onPressed: () => context.push(Routes.settingsNetworkMonitor),
+                    onPressed: null,
```

Comment:
If now this button is doing nothing, it might confuse users... maybe I'd be better to just remove it untill it is enabled?